client/components/search-sites/utils.ts

```diff
@@ -0,0 +1,7 @@
+import { get } from 'lodash';
+
+const matches = < T >( item: T, term: string, keys: ( keyof T )[] ) =>
+	keys.some( ( key ) => get( item, key, '' ).toLowerCase().indexOf( term ) > -1 );
```

Comment:
Given efforts to move away from `lodash` (p4TIVU-9Bf-p2), is some alternative to `get()` possible?